tailwind.config.js

```diff
@@ -43,24 +43,24 @@ module.exports = {
         'pulse-horizontal': 'pulse-horizontal 2s ease-in-out infinite',
       },
       colors: {
-        'blue-900': '#0d0030',
-        'blue-800': '#1a0259',
-        'blue-700': '#3a00d6',
-        'blue-600': '#4b61d5',
-        'blue-500': '#15a1fc',
-        'blue-400': '#bff5fa',
+        accent: '#15a1fc',
+        'accent-dark': '#0d0030',
+        'accent-light': '#bff5fa',
+        'accent-dimmed': '#3a00d6',
+        'accent-semi-dimmed': '#4b61d5',
+        'accent-background': '#1a0259',
 
-        orange: '#f1962e',
-        yellow: '#fed823',
-        pink: '#ff7bed',
-        green: '#01feb6',
+        primary: '#fed823',
+        secondary: '#ff7bed',
+        tertiary: '#01feb6',
         white: '#efecd6',
+        extra: '#f1962e',
```

Comment:
Let's get rid of this color, we don't use it

Reply:
Sure, but it's in the design, so I decided to leave it. But anyway, you're right